src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java

```diff
@@ -178,8 +178,12 @@ public void verifyRequestWithoutExecutorPostcodeReturnsError() {
     }
 
     @Test
-    public void verifyRequestCheckListAnswerEqualsYes() {
-        validatePostSuccess("solicitorPayloadNotifications.json", CHECKLIST_URL);
+    public void verifyRequestCheckListAnswerEqualsYesAndCheckQAState() {
+        ResponseBody body = validatePostSuccess("solicitorPayloadNotifications.json", CHECKLIST_URL);
+        JsonPath jsonPath = JsonPath.from(body.asString());
```

Comment:
COULD: vars that don't change once constructed are best marked as final, applies to most variables created in changeset

Reply:
potential to also use final var

Reply:
added